frontend/src/components/ui/question/QuestionBox.js

```diff
@@ -46,7 +59,9 @@ function QuestionBox() {
         });
     }
     fetchData();
-  }, []);
+    // To disable the lint warning
```

Comment:
I personally think that it's not a good practice to hide the warning. Maybe can try putting the dependency and see if there's any unexpected behaviour? If everything is normal, maybe just put it in the dependency array

Reply:
Yeap you're right. I was testing out previously if I placed difficulty in the dependency array, it causes some issues with retrieving the same question for both the matched users. Now, something must have been fixed such that I can include all the variables defined outside of the hook in the array. Thanks for checking this!